Tarea 1/tarea-1-index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <title>Monthly income calculator</title>
+    <link rel="stylesheet" href="file:///C:/Users/Seba/Documents/GitHub/5384/clase-5/Tarea%201/tarea-1.css">
+    <link rel="stylesheet" href="tarea-1.css">
+
+
+</head>
+
+<header>
+    <h1>Calculate your monthly income !</h1>
+    <h2>I've designed this tool gave you a quick response into how much you winning per month !</h2>
```

Comment:
I've designed this tool ``to give`` you a quick response into how much you winning per month!

Reply:
Fixed this typo.